src/layouts/job.astro

```diff
@@ -0,0 +1,44 @@
+---
+import MainLayout from '@/layouts/main.astro';
+import { Navbar } from '@/components/ui/navbar';
+import Footer from '@/components/ui/footer.astro';
+
+const { title, description } = Astro.props;
+---
+
+<MainLayout title={title} description={description}>
+  <Navbar client:load />
+  <article
+    class="prose prose-invert mx-auto w-11/12 py-9 lg:pt-20 xl:w-10/12 2xl:w-[1280px]"
+  >
+    <h1 class="m-0! text-center">{title}</h1>
+    <div class="py-8 text-center">
+      <p>
+        See <a href="https://handbook.shorebird.dev/"
+          >Shorebird's Public Handbook</a
+        > to learn more about us. Shorebird is a default-public company, so nearly
+        everything is open. You're always welcome to lurk on our <a
+          href="https://discord.gg/shorebird">Discord</a
+        >, read our <a href="https://github.com/shorebirdtech"
+          >GitHub issues/code/project plans</a
+        >, and <a href="https://handbook.shorebird.dev/"
+          >public company handbook</a
```

Comment:
Already linked above, can probably remove the second link.

Reply:
Done.